src/amber/controller/helpers/responders.cr

```diff
@@ -51,8 +61,9 @@ module Amber::Controller::Helpers
 
     private def requested_responses
       req_responses = Array(String).new
-
-      if (accept = context.request.headers["Accept"]?) && !accept.empty?
+      if (ext = request.path.match(Content::REGEX).try(&.[1]))
+        req_responses << Content::TYPE[ext]
+      elsif (accept = context.request.headers["Accept"]?) && !accept.empty?
         accepts = accept.split(";").first?.try(&.split(/,|,\s/))
```

Comment:
Make this regex `/,|,\s/` a constant that shows intent

Reply:
To me 4 regex characters are easier to read than tracking down the definition of a constant, but sure.